### PR TITLE
Add average rating to the daycare profile

### LINKED ISSUE #105: Add to the DayCare profile the average rating
## Expected Behavior

As a DayCare user when I open my profile I can see the average rating as part of the information
As a DogOwner user when I DayCare profile I can see the average rating of the DayCare as part of the information

## Current Behavior

There is no average rating

## Acceptance Criteria

The average rating is shown in the page
Tests added to test the new functionality

### PR DESCRIPTION
**Pull request by jeniaSakirko**
Signed-off-by: Jenia Sakirko <Jenia.Sakirko@cyberark.com>

### Desired Outcome

As a DayCare user when I open my profile I can see the average rating as part of the information
As a DogOwner user when I DayCare profile I can see the average rating of the DayCare as part of the information

### Implemented Changes

Implemented the new functionality:

1.  Under review there is new function and tests, the return is with two number after the dot
2. Under the daycare view, added the rating part to the context and added the rating part to the test
image

### Connected Issue/Story

Resolves #105 

### Definition of Done
See the AC in the relevant issue

#### Changelog

- [ ] The CHANGELOG has been updated
- [x] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [x] This PR includes new unit and integration tests to go with the code
  changes
- [ ] The changes in this PR do not require tests

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]()
- [x] This PR does not require updating any documentation

### CODE REVIEW

File: review/tests.py [hunk 1]
```diff
@@ -46,3 +46,25 @@ def test_get_review_by_daycare_id(self, number_of_reviews, create_dog_owner_user
...
+    @pytest.mark.parametrize('daycare_id, expected_rating', [(1, 2), (2, 4.67)])
+    def test_get_average_rating_for_test_data(self, daycare_id, expected_rating):
+        average_rating = Review.get_average_rating_by_daycare_id(daycare_id)
+
+        assert float(expected_rating) == average_rating
+
+    def test_get_average_rating_for_invalid_daycare(self):
+        average_rating = Review.get_average_rating_by_daycare_id(-1)
+
+        assert 0 == round(average_rating, 2)
```
Comment (Yuval-Vino) [inline review/tests.py:59]: Why use round here, in this case, shouldn't avrage_reveiw return 0 ?

**Review by Yuval-Vino — APPROVED** [review]
Great job @jeniaSakirko

**Review by ErezCohenn — APPROVED** [review]
GJ @jeniaSakirko 
One thing: I saw that there are 2 issues that are the same, #88 and #105 right? I think you should delete one of them.

**Comment by jeniaSakirko** [comment]
> GJ @jeniaSakirko One thing: I saw that there are 2 issues that are the same, #88 and #105 right? I think you should delete one of them.

Good one :) deleted #88

**Review by OfirMatasas — APPROVED** [review]
Good job!
For future pull requests, a picture of the new feature will be great :)

File: review/tests.py (hunk 1 above)
Comment (jeniaSakirko) [reply review/tests.py]: Fixed :)

**Pull request merged**